### PR TITLE
Maya: Fix tile rendering with Vray - OP-5566

### PR DESCRIPTION
**Pull request by tokejepsen**
## Changelog Description
Fixes tile rendering with Vray.

## Testing notes:
1. Setup tile render in Maya with Vray.
2. Publish and validate render is assembled correctly.

### CODE REVIEW

File: openpype/modules/deadline/plugins/publish/submit_maya_deadline.py [hunk 1]
```diff
@@ -518,6 +527,11 @@ def _tile_render(self, payload):
 
         instance.data["assemblySubmissionJobs"] = assembly_job_ids
 
+        # Remove config files to avoid confusion about where data is coming
+        # from in Deadline.
+        for config_file in config_files:
+            os.remove(config_file)
```
Comment (BigRoy) [inline openpype/modules/deadline/plugins/publish/submit_maya_deadline.py:533]: We're also passing a config to `aux_files` for the payload. How do these config files differ from those? Should we have better variable naming distinction?

File: openpype/modules/deadline/plugins/publish/submit_maya_deadline.py [hunk 2]
```diff
@@ -327,6 +327,11 @@ def _tile_render(self, payload):
         job_info = copy.deepcopy(payload_job_info)
         plugin_info = copy.deepcopy(payload_plugin_info)
 
+        # Force plugin reload for vray cause the region does not get flushed
+        # between tile renders.
+        if plugin_info["Renderer"] == "vray":
+            job_info.ForceReloadPlugin = True
```
Comment (BigRoy) [inline openpype/modules/deadline/plugins/publish/submit_maya_deadline.py:333]: This really sounds more like a Deadline Plugin issue which might be better solved there. Anyway, interesting catch!

File: openpype/modules/deadline/plugins/publish/submit_maya_deadline.py (hunk 1 above)
Comment (tokejepsen) [reply openpype/modules/deadline/plugins/publish/submit_maya_deadline.py:533]: Those are the same config files, but the Draft Tile Assembler plugin only gets the config file from `aux_files`, so having it in the plugin parameters or on disk, actually makes more confusion.

File: openpype/modules/deadline/plugins/publish/submit_maya_deadline.py (hunk 2 above)
Comment (tokejepsen) [reply openpype/modules/deadline/plugins/publish/submit_maya_deadline.py:333]: Yup, I agree but we are using built-in Draft Tile Assembler so we are at the hands of Thinkbox.

**Review by moonyuet — APPROVED** [review]
The code looks ok for me. Did some test with vray render and it doesn't have any error when submitting to deadline. The removal of config files doesn't affect the parameter of `aux-files` works in Job Info.
image

**Pull request merged**